src/Composer/StudioPlugin.php

```diff
@@ -43,8 +54,12 @@ public static function getSubscribedEvents()
      * This function configures Composer to treat all Studio-managed paths as local path repositories, so that packages
      * therein will be symlinked directly.
      */
-    public function registerStudioPackages()
+    public function registerStudioPackages(PreCommandRunEvent $event)
     {
+        if (! in_array($event->getCommand(), ['install', 'require', 'update', 'create-project'])) {
```

Comment:
Could probably use `$this->pluggedCommands` instead of hardcoded list.

Reply:
lol That was the point in first place...